webui/react/src/components/Searches/columns.ts

```diff
@@ -103,6 +103,13 @@ function getCellStateFromExperimentState(expState: CompoundRunState) {
   }
 }
 
+const EMPTY_CELL: GridCell = {
```

Comment:
`EMPTY_CELL` is the same as other `EMPTY_CELL`, so can we export the existing one instead of making new one?